fe2-android/app/src/main/res/layout/layout_election_display.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <data>
+        <import type="android.view.View" />
+        <import type="com.github.dedis.student20_pop.model.event.EventCategory" />
+        <variable
+            name="election"
+            type="com.github.dedis.student20_pop.model.Election" />
+        <variable
+            name="view_model"
+            type="com.github.dedis.student20_pop.detail.LaoDetailViewModel" />
+        <variable
+            name="eventCategory"
+            type="com.github.dedis.student20_pop.model.event.EventCategory" />
+    </data>
+
+    <LinearLayout
+        android:id="@+id/election_ll"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <LinearLayout
+            android:id="@+id/election_layout"
+            android:layout_width="186dp"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/election_title"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="15dp"
+                android:gravity="left"
+                android:text="Election : Election Name"
+                android:textSize="15dp"
```

Comment:
All the dimensions of the layouts have to be added to the values/dimens.xml file and then accessed as `"@dimen/dimension_name"`. Same thing for the strings, they have to be added to the values/strings.xml file and then accessed as `"@string/string_name"`